event/event.go

```diff
@@ -0,0 +1,181 @@
+// event is a package that models events that occur during the execution of a program
+package event
+
+// Event is an interface that custom events should implement.
+// It is empty for now because there are no general methods
+// required of events yet.
+type Event interface {
+}
+
+// KeyCharacter is an event that occurs when a character is actually typed on
+// the keyboard. This may be provided by an input method.
+type KeyCharacter struct {
+	// Code of the key pressed or released
+	Code int
```

Comment:
What kind of code value is inside? I guess we'd need an enum value.

Reply:
OK, I wixll make a type KeyCode, and enumerate all values. this will be a large file, so probably it is best to do this in a separate file keycode.go. If you agree with this idea then I will do this.

Reply:
Or, perhaps I should copy ebiten/key.go?

Reply:
https://github.com/hajimehoshi/ebiten/blob/master/internal/driver/keys.go I thought you could use `driver.Key` but unfortunately this is an internal package.

OK let's keep as int in this PR and leave TODO comment that we will define key enum and use it. I'll add key definitions to this event package later.

Reply:
OK!